lib/controller.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Controller do
+  def main(_args) do
+    run(:welcome_screen)
+  end
+
+  def run(prompt) when prompt != "Q" do
+    UserInterface.clear_screen()
+    parse_input(prompt)
+    UserInterface.line_break()
+
+    Messages.get_prompt(:menu)
+    |> UserInterface.get_input()
+    |> run()
+  end
+
+  def run("Q") do
+    execute_command("Q")
+  end
+
+  def parse_input(input) do
+    if Enum.member?(["G", "V", "I", :welcome_screen], input) do
```

Comment:
Maybe extract your menu options into an ENUM or something. that will allow you to change them as you please in one place without needing to change the rest of your code.

Reply:
Moreover, it might make more sense to have these be something more meaningful than single character strings in your code.